.github/workflows/create-release.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do you think it would be possible to run this after the CI job succeeds?

Reply:
Yeah I think this is a great suggestion, I've updated it so that it is now triggered on CI workflow, and will only create a release if the CI succeeds. That was based on my assumption that the CI action only gets run on merges to main, not for dev, etc.